sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -48,6 +48,19 @@ export interface AuthorizeRequestOptions {
     scopes: string[];
 }
 
+// @public
```

Comment:
A thought: do we actually need an additional policy for this feature? Couldn't we add an optional `auxiliaryCredentials` property to `bearerTokenAuthenticationPolicy` that handles this support if desired?

My understanding is that this policy is not sufficient to authenticate a request without a primary token.